ChatSecure/Classes/OTRAppDelegate.h

```diff
@@ -50,4 +50,10 @@
 /** Override this in subclass to use a different theme class */
 - (Class) themeClass;
 
+/** Override this in subclass to use a different conversation view controller class */
+- (Class) conversationViewControllerClass;
```

Comment:
@N-Pex I think we should move these to the `OTRTheme` class instead of on the app delegate